src/components/tests/QuickOpenModal.spec.js

```diff
@@ -163,4 +163,20 @@ describe("QuickOpenModal", () => {
       .simulate("change", { target: { value: "@someFunc" } });
     expect(filter).toHaveBeenCalledWith([], "someFunc", { key: "value" });
   });
+
+  test("Simple goto search query = :abc & searchType = goto", () => {
```

Comment:
Did you run this test locally? It doesn't seem to have a snapshot and normally this would modify the snapshot.

Reply:
I just added the snapshot right now.